interface/bigTree.h

```diff
@@ -1091,13 +1106,18 @@ class bigTree {
       fChain->SetBranchAddress("daughters_py_TauDown", &daughters_py_TauDown, &b_daughters_py_TauDown);
       fChain->SetBranchAddress("daughters_pz_TauDown", &daughters_pz_TauDown, &b_daughters_pz_TauDown);
       fChain->SetBranchAddress("daughters_e_TauDown", &daughters_e_TauDown, &b_daughters_e_TauDown);
-      fChain->SetBranchAddress("daughters_hasEES", &daughters_hasEES, &b_daughters_hasEES);
-      fChain->SetBranchAddress("daughters_EESshiftDM0up", &daughters_EESshiftDM0up, &b_daughters_EESshiftDM0up);
-      fChain->SetBranchAddress("daughters_EESshiftDM1up", &daughters_EESshiftDM1up, &b_daughters_EESshiftDM1up);
-      fChain->SetBranchAddress("daughters_EESshiftDM0dw", &daughters_EESshiftDM0dw, &b_daughters_EESshiftDM0dw);
-      fChain->SetBranchAddress("daughters_EESshiftDM1dw", &daughters_EESshiftDM1dw, &b_daughters_EESshiftDM1dw);
+      fChain->SetBranchAddress("daughters_hasFES", &daughters_hasFES, &b_daughters_hasFES);
+      fChain->SetBranchAddress("daughters_FESshiftDM0up", &daughters_FESshiftDM0up, &b_daughters_FESshiftDM0up);
+      fChain->SetBranchAddress("daughters_FESshiftDM1up", &daughters_FESshiftDM1up, &b_daughters_FESshiftDM1up);
+      fChain->SetBranchAddress("daughters_FESshiftDM0dw", &daughters_FESshiftDM0dw, &b_daughters_FESshiftDM0dw);
+      fChain->SetBranchAddress("daughters_FESshiftDM1dw", &daughters_FESshiftDM1dw, &b_daughters_FESshiftDM1dw);
```

Comment:
the branch name needs to be kept as it was; otherwise all changes looks good, so once this is fixed I'm happy to merge